server-plugins/notification-resources/src/index.ts

```diff
@@ -98,6 +98,43 @@ export async function OnBacklinkCreate (tx: Tx, control: TriggerControl): Promis
   return res
 }
 
+/**
+ * @public
+ */
+export async function OnDmCreate (tx: Tx, control: TriggerControl): Promise<Tx[]> {
+  const hierarchy = control.hierarchy
+  const ptx = tx as TxCreateDoc<DirectMessage>
+  const res: Tx[] = []
+
+  if (tx.createdBy == null) return []
+  if (!isDmCreationTx(ptx, hierarchy)) return []
+
+  const dm = (await control.findAll(ptx.objectClass, { _id: ptx.objectId }, { limit: 1 })).shift()
+  if (dm == null) return []
+
+  let dmWithPerson: Ref<Account> | undefined
+  for (const person of dm.members) {
+    if (person !== tx.createdBy) {
+      dmWithPerson = person
```

Comment:
What if this conversation with multiple users?

Reply:
I've added the checking for number of members, thanks